ExampleSwift/Podfile

```diff
@@ -5,6 +5,6 @@ target 'ExampleSwift - Pod' do
   use_frameworks!
 
   # Pods for Example
-  pod 'attentive-ios-sdk'
+  pod 'attentive-ios-sdk', '0.4.3-beta.1'
```

Comment:
Are we always going to want to define the exact version for the example app now? So in future version changes we'll want to bump this as well?

Reply:
I don't have a strong conviction either way. This change was to test out the beta functionality of cocoapods, but I can revert now that we know it works as expected.